dimod/constrained.py

```diff
@@ -618,6 +619,88 @@ def from_file(cls, fp: Union[BinaryIO, ByteString]) -> "ConstrainedQuadraticMode
 
         return cqm
 
+    def iter_violations(self, sample_like, *, skip_satisfied: bool = False, clip: bool = True,
+                        ) -> Iterator[Tuple[Hashable, Bias]]:
+        """Yield constraint labels and the amount the constraints are violated.
```

Comment:
"amount the constraints are violated" --> can we rephrase this? Something like perhaps, "Yield constraint labels and the size of any violations" to be colloquial or else "Yield constraint labels and the amplitude of any violations".
BTW, for the one liner I don't think you really need "label", this could just be "Yield violations for all constraints" and then more details in the subsequent paragraph.